### DISCUSSION TITLE
[Kavita] Adding an extra rating field that can be used to import ratings from external sources

### DISCUSSION THREAD

**Discussion by ShrinkWrapper** [Kareadita/Kavita#3198] [Ideas]
### Idea Description

Would be nice to have an extra rating field that can be populated with ratings from external sources kinda like a "critics rating", just like plex does with imdb and rotten tomatoes. 
In this instance it could be used together with Komf to source the ratings from example comick and add that into an extra field that could be displayed next to the built in user ratings.

Example:
![image](https://github.com/user-attachments/assets/af7ce473-3aa9-4189-8c60-25e40db83064)

### Idea Category

Feature Enhancement

### Duration of Using Kavita

5 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
This is a feature that already exists with Kavita+. Currently only works with manga through MAL and Anilist but Joe is looking to add more sources down the line

**Comment by majora2007** [comment]
As scare376 mentions, this is already possible with Kavita+. I don't want to muddy the Kavita application with external metadata connections, all of that falls under Kavita+ territory for me. If you have suggestions where there are APIs, I'm more than happy to explore them.

**Comment by ShrinkWrapper** [comment]
ah my bad, wasnt aware this was already a feature in Kavita+, guess that is my kickstart to get Kavita+, thanks and massive kudos on the last update, looks fantastic👌
